crates/swc_ecma_usage_analyzer/src/analyzer/mod.rs

```diff
@@ -234,7 +234,7 @@ enum RecursiveUsage {
 
 /// Analyzed info of a whole program we are working on.
 #[derive(Debug, Default)]
-pub(crate) struct ProgramData {
+pub struct ProgramData {
```

Comment:
Are you going to use this type directly?

Reply:
I prefer storing this type in `swc_ecma_minifier`, to reduce breaking changes.

Reply:
The current definition of `UsageAnalyzer` depends on it: `UsageAnalyzer<S = ProgramData>`. But I agree this makes more sense to live in `swc_ecma_minifier` for now so I'll move it and remove the default type.

From what I can tell the only breaking change is the `marks::Marks` type changing crates, and we can always re-export it from `swc_ecma_minifier`.

Reply:
Yeah, I think it's fine. Thank you!